app/build.gradle.kts

```diff
@@ -130,6 +131,9 @@ dependencies {
     implementation("androidx.compose.foundation:foundation")
     implementation("androidx.compose.material3:material3")
     implementation("androidx.compose.ui:ui-tooling-preview-android")
+    androidTestImplementation ("androidx.compose.ui:ui-test-junit4")
+    debugImplementation ("androidx.compose.ui:ui-test-manifest")
+    testImplementation ("androidx.compose.ui:ui-test-junit4:1.5.0")
```

Comment:
"androidx.compose.ui:ui-test-junit4" is part of "androidx.compose:compose-bom", so as I understand it setting a version number here would be incorrect, as "androidx.compose:compose-bom" defines the versions of all the subcomponents to make them work together well: https://mvnrepository.com/artifact/androidx.compose/compose-bom/2025.05.01